cli/commands/project.go

```diff
@@ -8,7 +8,7 @@ import (
 	"runtime"
 	"strings"
 
-	"github.com/phase2/rig/cli/commands/project"
+  "github.com/phase2/rig/cli/commands/project"
```

Comment:
Should be spaces, but mostly I just run go fmt and commit. Should I study that more?